### ISSUE TITLE
Function arguments aren't displayed in disassembly for ARM

### ISSUE THREAD

**Issue by gsingh93** [pwndbg/pwndbg#1554]
On x86, if a function is about to be called you see the argument values like this:
```
 ► 0x7ffff7fe32b3 <_start+3>             call   _dl_start                <_dl_start>
        rdi: 0x7fffffffbd90 ◂— 0x1
```

This doesn't seem to work on ARM. I haven't dug into this, but my guess is the root cause is the same as https://github.com/pwndbg/pwndbg/issues/1419, `bl` is only classified as a jump and not a call in `capstone`.

**Comment by mbrla0** [comment]
The rationale for the final fix (and choosing #1808 over #1797) is that, as of `5.0-post1`, Capstone made so that the workaround in #1797 is not needed anymore, which can be shown in the following way:

Versions:
```
$ git branch
* (HEAD detached at 5.0-post1)
  next
  
$ ./cstool -v
cstool for Capstone Disassembler, v5.0.0
Capstone build: x86=1 arm=1 arm64=1 mips=1 ppc=1 sparc=1 sysz=1 xcore=1 m68k=1 tms320c64x=1 m680x=1 evm=1 wasm=1 mos65xx=1 bpf=1 riscv=1 sh=1 tricore=1
```

AArch64 (#1797 has `BL` and `BLR` be considered calls):
```
$ aarch64-linux-gnu-as && aarch64-linux-gnu-objdump -d a.out
LABEL: BL LABEL
BLR X10

a.out:     file format elf64-littleaarch64


Disassembly of section .text:

0000000000000000 <LABEL>:
   0:   94000000        bl      0 <LABEL>
   4:   d63f0140        blr     x10

$ ./cstool -d arm64 00000094
 0  00 00 00 94  bl     #0
        ID: 68 (bl)
        op_count: 1
                operands[0].type: IMM = 0x0
                operands[0].access: READ
        Registers modified: lr
        Groups: call jump branch_relative

$ ./cstool -d arm64 40013fd6
 0  40 01 3f d6  blr    x10
        ID: 69 (blr)
        op_count: 1
                operands[0].type: REG = x10
                operands[0].access: READ
        Registers read: x10
        Registers modified: lr
        Groups: call jump branch_relative
```

ARM (#1797 has `BL` and `BLX` be considered calls):
```
$ arm-none-eabi-as && arm-none-eabi-objdump -d a.out
LABEL: BL LABEL
BLX R8

a.out:     file format elf32-littlearm


Disassembly of section .text:

00000000 <LABEL>:
   0:   ebfffffe        bl      0 <LABEL>
   4:   e12fff38        blx     r8

$ ./cstool -d arm feffffeb
 0  fe ff ff eb  bl     #0
        ID: 16 (bl)
        op_count: 1
                operands[0].type: IMM = 0x0
        Registers read: pc
        Registers modified: lr pc
        Groups: call branch_relative arm jump

$ ./cstool -d arm 38ff2fe1
 0  38 ff 2f e1  blx    r8
        ID: 17 (blx)
        op_count: 1
                operands[0].type: REG = r8
                operands[0].access: READ
        Registers read: pc r8
        Registers modified: lr pc
        Groups: call v5t jump
```

**Comment by gsingh93** [comment]
Should we also add a test for this?
